components/grid/index.stories.jsx

```diff
@@ -15,26 +17,61 @@ export default {
   },
 };
 
-export const Default = () => (
+const styleCol = {
+  border: '1px solid black',
+};
+
+export function Default() {
+  return (
   <Grid container>
     {[...Array(24)].map((item, index) => (
-      <Grid key={item}>
-        <p>{index + 1}</p>
+      <Grid key={item} style={styleCol}>
+        <Text variant="body2">{index + 1}</Text>
       </Grid>
     ))}
+    <Grid col={8} style={styleCol}>
+      <Text variant="body2">8</Text>
+    </Grid>
+    <Grid col={12} style={styleCol}>
```

Comment:
Codacy has a fix for the issue: [Expected indentation of 6 spaces but found 4.](https://app.codacy.com/gh/melishev/wuki/pullRequest?prid=8541441)
```suggestion
      <Grid col={12} style={styleCol}>
```